pkgbuild/Resources/preinstall

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+JDK_INSTALL_PATH="/Library/Java/JavaVirtualMachines"
+MKDIR=`which mkdir`
```

Comment:
Why this indirection?

Reply:
I've copied the Oracle and Corretto preinstall script here, I don't think that's strictly necessary but it shouldn't do any harm